pallets/handles/src/lib.rs

```diff
@@ -227,8 +230,14 @@ pub mod pallet {
 				None => {
 					next = 0;
 				},
-				Some(current) => {
-					next = current.checked_add(1).ok_or(Error::<T>::SuffixesExhausted)?;
+				Some((current, suffix_min)) => {
+					let current_suffix_min = T::HandleSuffixMin::get();
+					// if saved suffix min has changed, reset the suffix index
+					if current_suffix_min != suffix_min {
+						next = 0;
```

Comment:
If the sequence is reset to zero, won't this create duplicates?  If not, why?

Reply:
This assumes that we never have an overlapping min-max range.

If it overlaps ever, it doesn't matter what we do, we might duplicate as the shuffle will be different.

Example:

1-5 Shuffled (easy)
- 3
- 4
- 2
- 1
- 5

2-6 Shuffled
- 4
- 6
- 3
- 5
- 2